deploy/iso/minikube-iso/package/automount/minikube-automount

```diff
@@ -141,18 +114,6 @@ if [ -n "$BOOT2DOCKER_DATA" ]; then
     chown -R docker:docker /home/docker/.ssh
     rm -f '/home/docker/boot2docker, please format-me'
 
-    mkdir -p /mnt/$PARTNAME/var/lib/minikube
-    mkdir /var/lib/minikube
-    mount --bind /mnt/$PARTNAME/var/lib/minikube /var/lib/minikube
-
-    mkdir -p /mnt/$PARTNAME/var/lib/toolbox
-    mkdir /var/lib/toolbox
-    mount --bind /mnt/$PARTNAME/var/lib/toolbox /var/lib/toolbox
-
-    mkdir -p /mnt/$PARTNAME/var/lib/minishift
-    mkdir /var/lib/minishift
-    mount --bind /mnt/$PARTNAME/var/lib/minishift /var/lib/minishift
-
     ## make an env file for other services to discover this PARTNAME easier
     mkdir -p /var/run/minikube
```

Comment:
This brings up a great point: for systemd compatibility, please make `/var/run` a symlink to `/run` - otherwise we are introducing a behavioral change.

See: https://lwn.net/Articles/436012/